Ibistic.Public.OpenAirportData/Ibistic.Public.OpenAirportData/MemoryDatabase/AirportMemoryDatabase.cs

```diff
@@ -0,0 +1,77 @@
+﻿using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Ibistic.Public.OpenAirportData.MemoryDatabase
+{
+    public class AirportIataCodeDatabase
+    {
+        private readonly ConcurrentDictionary<string, Airport> _airports = new ConcurrentDictionary<string, Airport>();
+
+        public event EventHandler DatabaseExpired;
+
+        public DateTime DataUpdated { get; private set; }
+        public TimeSpan DatabaseExpiry { get; set; } = TimeSpan.FromDays(1);
+
+
+        public void AddOrUpdateAirports(IEnumerable<Airport> airports, bool removeOthers = false, bool ignoreAirportsMissingIataCode = false)
```

Comment:
Input validation is missing.

Reply:
Adding and adding method Clear() to empty the database.